src/wblib/figures/hifs.py

```diff
@@ -0,0 +1,26 @@
+"""Functions to work with IFS forecasts in healpix format."""
+import pandas as pd
+
+
+FORECAST_PUBLISH_LAG = "8h"
+FORECAST_PUBLISH_FREQ = "12h"
+
+
+def get_latest_forecast_issue_time(briefing_time: pd.Timestamp):
```

Comment:
I find it a bit confusing to name the variable `briefing_time` as we provide the briefing date without a time. In my head, the briefing time is always <date>T0900 local time...
Anyways, the function does what it should, so nothing to do here.